src/main/java/entity/Area.java

```diff
@@ -0,0 +1,46 @@
+package entity;
+
+import java.util.List;
+
+/**
+ * Created by admin on 15.03.2017.
+ */
+public class Area {
```

Comment:
Please, synchronize it with Venue entity, made in previous pull request. And change numOfPlaces to numberOfPlaces:)